src/main/kotlin/pet/project/app/dto/book/RequestUpdateBookDto.kt

```diff
@@ -0,0 +1,10 @@
+package pet.project.app.dto.book
+
+data class RequestUpdateBookDto(
```

Comment:
```suggestion
data class UpdateBookRequest(
```
fix all similar places